geode-core/src/test/java/org/apache/geode/cache/client/internal/AbstractOpTest.java

```diff
@@ -37,4 +43,39 @@ public void shouldBeMockable() throws Exception {
     assertThat(mockAbstractOp.processObjResponse(mock(Message.class), "string"))
         .isEqualTo(mockObject);
   }
+
+  @Test(expected = IOException.class)
+  public void processChunkedResponseShouldThrowIOExceptionWhenSocketBroken() throws Exception {
+    ChunkedMessage msg = mock(ChunkedMessage.class);
+    AbstractOp abstractOp = new AbstractOp(MessageType.PING, 0) {
```

Comment:
I think you can lean more on Mockito here instead of constructing an anonymous class. Something like:
```
AbstractOp abstractOp = spy(AbstractOp.class);
```
That should produce the same effect as your anonymous class.

Reply:
No. I tried your way. It did not work.